analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -226,7 +225,10 @@ class GoMod(
         val indirect: Boolean = false,
 
         @JsonProperty("Main")
-        val main: Boolean = false
+        val main: Boolean = false,
+
+        @JsonProperty("GoMod")
+        val goMod: String
```

Comment:
Could this be of type `File` right away?

Reply:
Probably it could, I prefer to keep it a String.